libs/checkpoint-postgres/pyproject.toml

```diff
@@ -13,6 +13,7 @@ python = "^3.9.0,<4.0"
 langgraph-checkpoint = "^1.0.1"
 orjson = ">=3.10.1"
 psycopg = {extras = ["binary"], version = ">=3.1.19"}
```

Comment:
I don't think this should be installing the binary extra out of the box, can conflict with user's dependency on one of the other psycopg variants